src/main/kotlin/com/openlattice/postgres/RowSecurityPolicy.kt

```diff
@@ -0,0 +1,20 @@
+package com.openlattice.postgres
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.client.serialization.SerializationConstants
+import java.util.*
+
+data class RowSecurityPolicy(
```

Comment:
Can you also add some javadoc for this class pls?

Reply:
yep will do!